pypeit/core/datacube.py

```diff
@@ -5,15 +5,146 @@
 .. include:: ../include/links.rst
 """
 
+import inspect
+
 from astropy import wcs, units
 from astropy.coordinates import AltAz, SkyCoord
+from astropy.io import fits
 import scipy.optimize as opt
 from scipy.interpolate import interp1d
 import numpy as np
 
 from pypeit import msgs
 from pypeit.core.procimg import grow_masked
-from pypeit.core import fitting, coadd
+from pypeit.core import coadd
+from pypeit.spectrographs.util import load_spectrograph
+from pypeit import datamodel
+from pypeit import io
+
+
+class DataCube(datamodel.DataContainer):
+    """
+    DataContainer to hold the products of a datacube
+
+    See the datamodel for argument descriptions
+
+    Args:
```

Comment:
This does not look up to date, and shouldn't the args be listed under the __init__.